src/Tooltip.js

```diff
@@ -110,10 +110,12 @@ const defaultProps = {
 }
 
 const Tooltip = ({ children, ...props }) => {
+  const { align, ..._props } = props
```

Comment:
I think we should avoid destructuring props twice – any reason not to define `align` in the line above?